### PR TITLE
[TOPI][CUDA] schedule for group_conv2d

### PR DESCRIPTION
**Pull request by vinx13**

### CODE REVIEW

**Comment by vinx13** [comment]
@merrymercy @masahi @kevinthesun please help review

File: topi/python/topi/cuda/group_conv2d_nchw.py [hunk 1]
```diff
@@ -321,8 +321,124 @@ def schedule_group_conv2d_NCHWc_int8(cfg, s, output):
...
+        s[load].bind(tz, tvm.thread_axis("threadIdx.z"))
+        s[load].bind(ty, tvm.thread_axis("threadIdx.y"))
+        s[load].bind(tx, tvm.thread_axis("threadIdx.x"))
+
+    # unroll
+    s[output].pragma(kernel_scope, 'auto_unroll_max_step', cfg['auto_unroll_max_step'].val)
+    s[output].pragma(kernel_scope, 'unroll_explicit', cfg['unroll_explicit'].val)
+
+    N, CO, OH, OW = get_const_tuple(output.shape)
+    _, KH, KW, CI = get_const_tuple(kernel.shape)
```
Comment (masahi) [inline topi/python/topi/cuda/group_conv2d_nchw.py]: Is this layout correct? Shouldn't it be CO, group, KH, KW?

Comment (vinx13) [reply topi/python/topi/cuda/group_conv2d_nchw.py]: nice catch!

**Comment by masahi** [comment]
thanks @vinx13, this is merged.

**Pull request merged**